lightning/src/ln/outbound_payment.rs

```diff
@@ -687,15 +687,19 @@ pub(super) struct SendAlongPathArgs<'a> {
 
 pub(super) struct OutboundPayments {
 	pub(super) pending_outbound_payments: Mutex<HashMap<PaymentId, PendingOutboundPayment>>,
-	pub(super) awaiting_invoice_flag: AtomicBool,
-	pub(super) retry_lock: Mutex<()>,
+	awaiting_invoice_flag: AtomicBool,
```

Comment:
Let's drop `_flag` from the name as it is implied from the type.

Reply:
Makes sense!
Remove the `_flag` suffix in [420ed56](https://github.com/lightningdevkit/rust-lightning/commit/420ed565bf1fce14e96c02e05fa06a7b49e3a3ec)
Thanks!